main/pipeline/pipeline.cc

```diff
@@ -1544,4 +1546,27 @@ void printUntypedBlames(const core::GlobalState &gs, const UnorderedMap<long, lo
 #endif
 }
 
+std::unique_ptr<core::GlobalState> copyForSlowPath(const core::GlobalState &from, const options::Options &opts) {
+#ifdef SORBET_REALMAIN_MIN
+    // It is impossible to enter this function if SORBET_REALMAIN_MIN is defined, as the only entry should be from a
+    // running LSP session.
+    Exception::raise("LSP is disabled in sorbet-orig for faster builds");
+#else
```

Comment:
I don't think we need this guard, do we?

We typically only add these guards when there's something that would fail to compile in the orig target, but it seems that this compiles fine even in the orig target.
```suggestion
```

Reply:
I think it's fine to remove, especially since it shouldn't be reachable if `SORBET_REALMAIN_MIN` is defined--the only path would be through LSP which is not built in that case.